2/contrib/s2i/run

```diff
@@ -56,8 +58,10 @@ function create_jenkins_config_from_templates() {
             elif [[ "${target_path}" == "${image_config_dir}/credentials.xml" ]]; then
                 create_jenkins_credentials_xml
             else
-                # Allow usage of environment variables in templated files, e.g. ${DOLLAR}MY_VAR is replaced by $MY_VAR
-                DOLLAR='$' envsubst < "${template_path}" > "${target_path}"
+                # 1) Allow usage of environment variables in templated files, e.g. ${DOLLAR}MY_VAR is replaced by $MY_VAR;
+		# 2) have to fake out envsubst to wrt internal java class reference hudson.model.UpdateCenter$CoreUpdateMonitor
+		# where we are disabling jenkins core update monitor because of CPU churn on initial login
+                CoreUpdateMonitor="\$CoreUpdateMonitor" DOLLAR='$' envsubst < "${template_path}" > "${target_path}"
```

Comment:
this block isn't applicable to config.xml.tpl anyway.

Reply:
Yeah I saw that in testing, but was uncertain exactly what path this did in fact cover.  Is it for injecting templates *OTHER* than the config.xml and credentials.xml?

Anyway, if there was anyway this could still impact config.xml, it would be covered, and shouldn't impact the other cases.

But sure, based on the clarification, I can remove if it makes sense.

Reply:
>  Is it for injecting templates OTHER than the config.xml and credentials.xml?

yes.  and i'm not sure there's really a great reason there being special switches for each file, but yes.

> But sure, based on the clarification, I can remove if it makes sense.

overall i'd rather move to a common ${DOLLAR} based approach for all *tpl substitution. assuming you can sort out why it wasn't working for you earlier.